boot/usage.ml

```diff
@@ -9,7 +9,7 @@
 (************************************************************************)
 
 let version () =
-  Printf.printf "The Rocq Proof Assistant, version %s\n" Coq_config.version;
+  Printf.printf "The Rocq Prover, version %s\n" Coq_config.version;
```

Comment:
This will likely be harder to merge. Expect need for compcert and fiat-parsers overlay. You may want to extract that into its own PR.

Reply:
:cry:

Reply:
Moved to #20052.

Reply:
Sorry, I should have noticed that in #20027